env.dev.pg.base

```diff
@@ -192,115 +179,20 @@ PLUGIN_HANDLER_BIOM_PROJECT_IDS=MicrobiomeDB
 # App Database Connections  #
 # # # # # # # # # # # # # # #
 
-DB_CONNECTION_ENABLED_AMOEBA=false
```

Comment:
Can we remove these "enabled" flags from the Oracle version too?  I thought they were required?

Reply:
Sure, they're no longer required since I removed the extra connections from the compose.